CONTRIBUTING.md

```diff
@@ -23,9 +23,7 @@ This the default behavior.
 
     `mvn install -Dtest.github.useProxy -Dtest=WireMockStatusReporterTest`
 
-4. The above should report no test failures and include the following console output:
-
-    `WireMockStatusReporterTest: GitHub proxying and user auth correctly configured for user login: <your login>`
+4. The above should report no test failures
```

Comment:
Hm, I wonder why that was changed.  I'll have to look into that.  Could you please file an issue for this and reverse this change?

Reply:
👉 #785